nginx/etc/nginx/conf.d/tiler.conf

```diff
@@ -15,7 +15,7 @@ server {
 	include /etc/nginx/includes/tiler-security-headers.conf;
 
 	location ~ /tiles/(.*) {
-		return 301 $scheme://$http_host/$1$is_args$args;
```

Comment:
Can you add a quick note above this location block that says something about this route effectively being deprecated?